lib/controller_patches.rb

```diff
@@ -13,9 +13,13 @@ def house_rules; end
     private
 
     def set_history
-      @history ||= HelpPageHistory.new(
-        lookup_context.find_template("#{controller_path}/#{action_name}")
-      )
+      # Only set history if a template exists for this action
+      begin
+        template = lookup_context.find_template("#{controller_path}/#{action_name}")
+        @history ||= HelpPageHistory.new(template)
+      rescue ActionView::MissingTemplate
+        # No template for this action, skip setting history
+      end
```

Comment:
i think ruby let's you imply the `begin`
```suggestion
      # Only set history if a template exists for this action
      template = lookup_context.find_template("#{controller_path}/#{action_name}")
      @history ||= HelpPageHistory.new(template)
    rescue ActionView::MissingTemplate
      # No template for this action, skip setting history
```